apps/s_time.c

```diff
@@ -432,6 +378,16 @@ static SSL *doConnection(SSL *scon, const char *host, SSL_CTX *ctx)
             SSL_free(serverCon);
         return NULL;
     }
+#if defined(SOL_SOCKET) && defined(SO_LINGER)
+    else {
```

Comment:
Some even "hate the else after return" :-) But to me, in this case, it's really `else` in #if. I'm not actually a stranger to such construct (I mean I've used it myself), but it's always "unless actually necessary". And it's not actually necessary in this case...